mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
I'm not sure I trust myself to get the syntax, but I'll attempt to write a more pointed popup for this.